### PR TITLE
text.last-index-of implemented

### PR DESCRIPTION
**Pull request by EugeneDar**
https://github.com/objectionary/eo-strings/issues/43

### CODE REVIEW

**Comment by EugeneDar** [comment]
@Graur as I remember, you also encountered a problem with tests falling only on Windows. Is it possible that there is a similar problem here?

**Comment by Graur** [comment]
> @Graur as I remember, you also encountered a problem with tests falling only on Windows. Is it possible that there is a similar problem here?

@EugeneDar Yes, this is the same problem, I will fix it as soon as I can

File: src/test/eo/org/eolang/txt/text-tests.eo [hunk 1]
```diff
@@ -153,6 +153,21 @@
...
+[] > text-last-index-of-2
+  assert-that > @
+    ((text "some text here").last-index-of "  here")
+    $.equal-to -1
+
+[] > text-last-index-of-3
+  assert-that > @
+    ((text "abc abcd").last-index-of "abc")
+    $.equal-to 4
+
```
Comment (Graur) [inline src/test/eo/org/eolang/txt/text-tests.eo:170]: @EugeneDar Please, add tests with corner cases and Unicode characters

**Review by Graur — CHANGES_REQUESTED** [review]
@EugeneDar please, check below:

**Comment by EugeneDar** [comment]
@Graur please, have a look again

**Review by Graur — APPROVED** [review]
@EugeneDar Looks good for me
@yegor256 Please, take a look

**Comment by EugeneDar** [comment]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor merge
